app/views/organisations/students/show.html.erb

```diff
@@ -4,11 +4,16 @@
       <div class="mt-4">
         <a class="bg-gray-200 px-3 py-1 text-gray-600 rounded-xl text-sm hover:text-primary-500 hover:bg-primary-50 focus:outline-none focus:text-primary-500 focus:bg-primary-50 focus:ring-1 focus:ring-focusColor-500" href="<%= students_organisation_cohort_path(@presenter.organisation, @presenter.cohort, @presenter.filters_in_url) %>"><%= t('shared.back_link') %></a>
       </div>
-      <div class="mt-5 flex items-center">
-        <div class="w-11">
-          <div data-re-component="Avatar" data-re-json="<%= { name: @presenter.student.name }.to_json %>"></div>
+      <div class="flex flex-col sm:flex-row items-start lg:items-center mt-5 lg:space-x-3 overflow-x-auto">
+        <div class="flex items-center">
+          <div class="w-11">
+            <div data-re-component="Avatar" data-re-json="<%= { name: @presenter.student.name }.to_json %>"></div>
+          </div>
+          <h1 class="text-xl md:text-2xl font-semibold ps-4 whitespace-normal"><%= @presenter.student.name %></h1>
+        </div>
+        <div class="text-sm flex-shrink bg-gray-200 bg-opacity-25 lg:bg-opacity-50 px-1.5 py-1 mt-2 lg:mt-0 text-gray-600 rounded-lg font-semibold whitespace-nowrap">
```

Comment:
1. `flex-shrink` is replaced with `shrink` in **tailwind 3.X**
2. Why are we changing the opacity of the `background color` on the basis of screen size?
3. Since the parent is already a flex container, we can add` gap-*` there and remove the margin from this element.